sources/editor/Stride.Assets.Presentation/Templates/ModelFromFileTemplateGenerator.cs

```diff
@@ -57,8 +57,9 @@ protected override async Task<bool> PrepareAssetCreation(AssetTemplateGeneratorP
             if (files == null)
                 return true;
 
-            var showDeduplicateMaterialsCheckBox = files.Any(x => ImportAssimpCommand.IsSupportingExtensions(x.GetFileExtension()));
-            var showFbxDedupeNotSupportedWarning = showDeduplicateMaterialsCheckBox && files.Any(x => ImportFbxCommand.IsSupportingExtensions(x.GetFileExtension()));
+            var showDeduplicateMaterialsCheckBox = files.Any(x => ImportThreeDCommand.IsSupportingExtensions(x.GetFileExtension()));
+         /*   var showFbxDedupeNotSupportedWarning = showDeduplicateMaterialsCheckBox && files.Any(x => ImportFbxCommand.IsSupportingExtensions(x.GetFileExtension()));
+          */
```

Comment:
Remove code instead of commenting it out.

Reply:
Done!